backend/alembic/versions/a8b9c0d1e2f3_add_linked_group_id_to_tasks.py

```
@@ -0,0 +1,356 @@
+        ).fetchall()
+
+        # Filter tasks with linked_group in Python for SQLite
+        filtered_result = []
+        for row in result:
+            try:
+                task_json = json.loads(row[1]) if isinstance(row[1], str) else row[1]
+                linked_group = task_json.get("spec", {}).get("linked_group")
+                if linked_group:
+                    filtered_result.append((row[0], linked_group))
+            except Exception:
+                pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)